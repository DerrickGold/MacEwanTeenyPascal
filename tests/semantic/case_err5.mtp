(*Try duplicate case values in the same case*)
const test := 1;
var a, b : integer;


begin

	case a + b of
	1, test:
		a := b

	end


end.