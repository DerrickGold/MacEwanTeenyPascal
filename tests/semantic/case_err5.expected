
Semantic Error [line 9]: value: "test"
	Duplicate case detected. All cases must be unique.
